src/main/java/io/github/moulberry/notenoughupdates/mixins/MixinGuiTextField.java

```diff
@@ -80,7 +80,7 @@ public void writeText_stringStack(String string, CallbackInfo ci, String string2
 		}
 	}
 
-	@Inject(method = "deleteFromCursor", at = @At(value = "INVOKE", target = "Lcom/google/common/base/Predicate;apply(Ljava/lang/Object;)Z"), locals = LocalCapture.PRINT)
+	@Inject(method = "deleteFromCursor", at = @At(value = "INVOKE", target = "Lcom/google/common/base/Predicate;apply(Ljava/lang/Object;)Z"), locals = LocalCapture.PRINT, remap = false)
```

Comment:
are you sure that mixin complains about remap and not about the localcapture

Reply:
shouldn't be a remap issue as the mixin is of a net.minecraft class

Reply:
warning: Cannot find method mapping for @At(INVOKE.<target>) 'Lcom/google/common/base/Predicate;apply(Ljava/lang/Object;)Z'

Reply:
also the LocalCapture.PRINT shouldn't be there anymore, prolly forgot to remove it

Reply:
that would still be the wrong place to remove the remap (should be remap=false in the At, not in the Inject

Reply:
in its current form this would just not remap the `deleteFromCursor` which seems like a bad idea

Reply:
in general: i cannot repro this warning at all